rust/src/domains/polars/expr/mod.rs

```diff
@@ -41,11 +41,13 @@ pub enum ExprContext {
 }
 
 impl ExprContext {
-    fn get_plan(&self, val: &(DslPlan, Expr)) -> DslPlan {
-        let (lp, expr) = val.clone();
-        let frame = LazyFrame::from(lp);
+    /// # Proof Definition
+    /// Given a context (`self`), logical plan (`lp`) and expression,
```

Comment:
Not particular to this PR, but would `plan` be more readable than `lp`, at the cost of two more characters?